apps/mobile/src/utils/transaction.ts

```diff
@@ -45,6 +45,34 @@ export const createAndSend: CreateAndSendFunction = async (
 	} else if (payload.network == Networks.tezos) {
 		// TODO: implement transfer tezos
 		console.log('hello tezos');
+	} else if (payload.network === Networks.aptos) {
+		let privateKey;
+		try {
+			privateKey = await utils.getPrivateKey(Networks.aptos, passcode);
+		} catch {
+			res.responseCode = ResponseCode.WRONG_PASSCODE;
+			return res;
+		}
+
+		const isCoinTransaction = !('creator' in transaction);
+
+		try {
+			if (isCoinTransaction) {
+				res.signatureString = await aptosHandlers.handleTransferCoin(
+					privateKey,
+					JSON.stringify(transaction),
```

Comment:
in mobile, we can prevent serialize transaction object over calling kenel handler